plugins/cache.rst

```diff
@@ -33,22 +33,78 @@ control headers from the server as specified in :rfc:`7234`. It needs a
         [$cachePlugin]
     );
 
-By default, responses with no cache control headers are not cached. If you want
-a default cache lifetime if the server specifies no ``max-age``, use::
+Options
+-------
+
+The third parameter to the ``CachePlugin`` constructor takes an array of options. The plugin has 3 options you can
+configure. Their default values and meaning is described by the table below.
+
++---------------------------+---------------+------------------------------------------------------+
+| Name                      | Default value | Description                                          |
++===========================+===============+======================================================+
+| ``default_ttl``           | ``null``      | The default max age of a Response                    |
++---------------------------+---------------+------------------------------------------------------+
+| ``respect_cache_headers`` | ``true``      | Whatever or not we should care about cache headers   |
++---------------------------+---------------+------------------------------------------------------+
+| ``cache_lifetime``        | 30 days       | The minimum time we should store a cache item        |
++---------------------------+---------------+------------------------------------------------------+
+
+.. note::
+
+    A HTTP response may have expired but it is still in cache. If so, headers like ``If-Modified-Since`` and
+    ``If-None-Match`` are added to the HTTP request to allow the server answer with 304 status code. When
+    a 304 response is recieved we update the CacheItem and save it again for at least ``cache_lifetime``.
+
+Using these options together you may control the how your cached responses behave. By default, responses with no
+cache control headers are not cached. If you want a default cache lifetime if the server specifies no ``max-age``, use::
 
     $options = [
         'default_ttl' => 42, // cache lifetime time in seconds
     ];
 
-You can also tell the plugin to completely ignore the cache control headers
-from the server and force caching for the default time to life. Note that in
-this case, ``default_ttl`` is required::
+If you tell the plugin to completely ignore the cache control headers from the server and force caching the response
+for the default time to life. The options below will cache all responses for one hour::
 
     $options = [
         'default_ttl' => 3600, // cache for one hour
         'respect_cache_headers' => false,
     ];
 
+Be ware of null values
+``````````````````````
+
+Setting null to the options ``cache_lifetime`` or ``default_ttl`` means "Store this as long as you can (forever)".
+This could be a great thing when you requesting a pay-per-request API (eg. GoogleTranslate).
```

Comment:
You are correct. I guess the `default_ttl=null` has changed it's behavior since the initial release. 

But if there is no headers that enforce caching we should not cache the response for 10 minutes.. We should default the ttl to 0. (No cache).